packages/babel-preset-react-app/package.json

```diff
@@ -21,13 +21,13 @@
     "test.js"
   ],
   "dependencies": {
-    "@babel/core": "7.7.5",
-    "@babel/plugin-proposal-class-properties": "7.7.4",
-    "@babel/plugin-proposal-decorators": "7.7.4",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "7.7.4",
-    "@babel/plugin-proposal-numeric-separator": "7.7.4",
+    "@babel/core": "7.8.3",
+    "@babel/plugin-proposal-class-properties": "7.8.3",
+    "@babel/plugin-proposal-decorators": "7.8.3",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "7.8.3",
```

Comment:
I believe this is handled by preset-env in Babel 8. Same with Optional Chaining plugin.

Reply:
It was all here before, so I'm not going to touch it unless a facebook employee says so.

Reply:
Feel free to leave it as you wish.

This is new to `@babel/preset-env` version 7.8.x.

https://babeljs.io/blog/2020/01/11/7.8.0#ecmascript-2020-default-support-10811-https-githubcom-babel-babel-pull-10811-10817-https-githubcom-babel-babel-pull-10817-10819-https-githubcom-babel-babel-pull-10819-10843-https-githubcom-babel-babel-pull-10843

In the blog announcing the release of babel 7.8.0 they specifically mention removing direct reference to optional chaining and nullish coalescing because they are integrated into the latest `@babel/preset-env`.

```diff
{
  "presets": [
    ["@babel/env", { "targets": ["last 2 versions"] }]
  ],
  "plugins": [
-   "@babel/proposal-optional-chaining",
-   "@babel/proposal-nullish-coalescing-operator"
  ] 
}
```

That's because those packages are [now set as hard dependencies](https://github.com/babel/babel/blob/master/packages/babel-preset-env/package.json) and don't need to be manually added.

Reply:
Will remove these in a follow-up PR. Thanks @heygrady.